controllers/avatars.js

```diff
@@ -0,0 +1,68 @@
+const { stat, mkdir } = require("fs/promises");
+const multer = require("multer");
+const jimpConvert = require("../models/jimp");
+const path = require("path");
+const { User } = require("../models/schema");
+
+const dirname = path.dirname(__dirname);
+const avatarsFolder = path.join(dirname, "public", "avatars");
+let pathToTheFile;
+
+stat(avatarsFolder).catch(() => mkdir(avatarsFolder, { recursive: true }));
+
+const storage = multer.diskStorage({
+  destination: function (_, __, cb) {
```

Comment:
Może uzywaj tego samego stylu jesli nie ma potrzeby zeby zmieniac? Bo tu nie odwołujesz się do this.